runtime/src/cost_tracker.rs

```diff
@@ -89,51 +98,33 @@ impl CostTracker {
         self.vote_cost_limit = vote_cost_limit;
     }
 
-    pub fn would_transaction_fit(
-        &self,
+    pub fn try_add(
+        &mut self,
         transaction: &SanitizedTransaction,
         tx_cost: &TransactionCost,
-    ) -> Result<(), CostTrackerError> {
-        self.would_fit(
-            &tx_cost.writable_accounts,
-            tx_cost.sum(),
-            tx_cost.account_data_size,
-            transaction,
-        )
+    ) -> Result<u64, CostTrackerError> {
+        self.would_fit(tx_cost)?;
+        self.pending_transactions
+            .insert(*transaction.signature(), tx_cost.clone());
```

Comment:
It would be nice to avoid this clone.  `TransactionCost`s are large

Reply:
This has already been resolved, the clone is gone